src/robotFactory.js

```diff
@@ -1,10 +1,67 @@
 'use strict';
 
-class BaseRobot {}
+class BaseRobot {
+  constructor(name, weight, coords, chipVersion) {
+    this.name = name;
+    this.weight = weight;
+    this.chipVersion = chipVersion;
 
-class FlyingRobot {}
+    this.coords = coords === undefined || coords.x === undefined ? {
+      x: 0,
+      y: 0,
+    } : coords;
+  }
 
-class DeliveryDrone {}
+  goForward(step = 1) {
+    this.coords.y += step;
+  }
+
+  goBack(step = 1) {
+    this.coords.y -= step;
+  }
+
+  goRight(step = 1) {
+    this.coords.x += step;
+  }
+
+  goLeft(step = 1) {
+    this.coords.x -= step;
+  }
+  getInfo() {
```

Comment:
```suggestion
  }
  
  getInfo() {
```
Коли відмежовуєш метди пустими рядками - виглядає краще